.github/workflows/maven-verify.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           cache: 'maven'
 
       - name: Set up Maven
-        run: mvn --errors --batch-mode --show-version wrapper:wrapper "-Dmaven=3.8.7"
+        run: mvn --errors --batch-mode --show-version wrapper:wrapper "-Dmaven=3.9.0"
```

Comment:
Is this the version which is used to run the actual ITs or just compile the project?

Reply:
in this build for both - but ok test is only for 4.x